WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -0,0 +1,618 @@
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.deletePost(page)
+                })
+            } else if filter == .Published {
+                alertController.addActionWithTitle(viewButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.viewPost(page)
+                })
+                
+                alertController.addActionWithTitle(draftButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.draftPage(page)
+                })
+                
+                alertController.addActionWithTitle(trashButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.deletePost(page)
+                })
+            } else {
+                alertController.addActionWithTitle(viewButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+
+                    strongSelf.viewPost(page)
+                })
+                
+                alertController.addActionWithTitle(publishButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.publishPost(page)
+                })
+                
+                alertController.addActionWithTitle(trashButtonTitle, style: .Default, handler: { [weak self] (action) in
+                    guard let strongSelf = self,
+                        let page = strongSelf.pageForObjectID(objectID) else {
+                            return
+                    }
+                    
+                    strongSelf.deletePost(page)
+                })
+            }
+        }
+        
+        WPAnalytics.track(.PostListOpenedCellMenu, withProperties: propertiesForAnalytics())
+        
+        if UIDevice.isPad() {
```

Comment:
This logic is inverted from the original it should be "!UIDevice.isPad()".

I think even better will be to get rid of this IF and let the system decide how to present this, it should work ok.